style.css

```diff
@@ -0,0 +1,376 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Quicksand:wght@300;400;500;600;700&family=Ubuntu+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap");
+
+/* ------------------- */
+/* Resets              */
+/* ------------------- */
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+p,
+figure,
+picture {
+  margin: 0;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  font-weight: 400;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+/* ------------------- */
+/* Custom properties   */
+/* ------------------- */
+
+:root {
+  --clr-dark: 0, 22, 50;
+  --clr-grey: 189, 193, 197;
+  --clr-grey-light: 211, 211, 211;
+  --clr-accent: 85, 172, 238;
+  --clr-white: 255, 255, 255;
+
+  --ff-global: "poppins", sans-serif;
+}
```

Comment:
Nice! I found the variables very useful as well. I was unsure about where to put them, why the `:root` selector?

Reply:
i believe the root selector is selecting the html tag but just with higher specificity. i was under the impression it was standard to use root so thats why i used it haha